src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -45,6 +45,13 @@ internal sealed partial class Http2Connection : HttpConnectionBase, IDisposable
         private long _idleSinceTickCount;
         private int _pendingWriters;
         private bool _lastPendingWriterShouldFlush;
+        private int _freeStreamSlots;
```

Comment:
Naming-wise, it's very confusing that in one place `_freeStreamSlots` is declared as an `int` and in another `_freeStreamSlots` is declared as a `SemaphoreSlim`.

Reply:
Renamed.